apps/meteor/app/lib/server/functions/sendMessage.ts

```diff
@@ -212,14 +213,71 @@ export function prepareMessageObject(
 	}
 }
 
+const parseFilesIntoMessageAttachments = async (
```

Comment:
Great, this is right, I'd only move this function to the same place of `parseFileIntoMessageAttachments`

Reply:
Thank you sir for your feedback!

Currently I've renamed the function to `parseMultipleFilesIntoAttachments` and moved its location to `sendFileMessage` . or else we could update `parseFileIntoMessageAttachments` to handle multiple files also. Which approach do you think is better?

Reply:
`parseMultipleFilesIntoAttachments` sounds good